tests/test_datauri.py

```diff
@@ -38,6 +38,21 @@ def test_parse_base64_with_missing_padding():
     assert parsed.data == b'any carnal pleasur'
 
 
+# https://github.com/eclecticiq/python-data-uri/issues/7
+@pytest.mark.parametrize('data', [
+    'data:text/plain;base64, YW55IGNhcm5hbCBwbGVhcw',
+    'data:text/plain;base64,YW55IGNhcm5hbCBwbGVhcw  ',
+    'data:text/plain;base64,YW55IGNhcm 5hbCBwbGVhcw',
+    'data:text/plain;base64,\tYW55IGNhcm5hbCBwbGVhcw',
+    'data: text/plain; base64,YW55IGNhcm5hbCBwbGVhcw',
```

Comment:
this...

Reply:
What? Beautiful?

Reply:
oh oops forgot. i meant to say that the spaces in the actual data are fine to ignore, but not sure about those in the header... is that even allowed/common?

Reply:
IDK. I decided to resolve everything if it's possible if `strict=False`. Be as smart as possible. It's like a totally different way to work. As you said in #8 "code should either be strict or loose"

Reply:
yeah, since spaces in the data part actually seems to happen in html... and i was wondering whether that can also be in the header part. i guess the answer is yes, and there is no harm in just ignoring these spaces here. :+1: